### ISSUE TITLE
Register Xperience Identity features at startup

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#11]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [x] Developer

### Needs what

To register the Xperience implementation of ASP.NET Identity during the app startup.

### Why

To let users sign in using local Xperience user accounts.

### Issue expands upon

This issue expands upon another issue(s): #10

### Blockers

This issue is blocked by the following issue(s): #`issue number`
